trillian/ctfe/storage/storage.go

```diff
@@ -0,0 +1,17 @@
+package storage
```

Comment:
Similar to another PR I commented on, this needs the big header with copyright, and a package docstring.

Reply:
Add the license header and package doc.